src/Forms/SearchableDropdownTrait.php

```diff
@@ -598,6 +597,9 @@ public function performReadonlyTransformation()
         $field->setSource($this->sourceList);
         $field->setReadonly(true);
 
+        // Remove the text "Type to search..." on a read-only field
+        $this->setUseDynamicPlaceholder(false);
```

Comment:
```suggestion
        $field->setUseDynamicPlaceholder(false);
```
Shouldn't this be called on the field we're returning?

Reply:
Yep, it should. I suspect it won't work because we are now using a different field type, so I'm going to keep playing with this 'til my test passes 🤔